### PR TITLE
fix issue with en-IE currency

### PR DESCRIPTION
**Pull request by Blackbaud-WesGabbard**
Ran into issue when trying to format currency for Ireland. When setting culture as en-IE was getting the following error:

`Currency position should be among ["prefix", "infix", "postfix"]`

Looking at the en-IE.js file in the languages folder, it is missing a property defining the position of the currency symbol. Added this to be prefix since Euro symbols go in front. Updated the test to account for this.

### CODE REVIEW

**Comment by Blackbaud-WesGabbard** [comment]
@BenjaminVanRyseghem, just checking in on this PR and see if there is anything else needed on my end. Thanks.

**Comment by Blackbaud-WesGabbard** [comment]
@BenjaminVanRyseghem checking in on this PR, is there any reason this can't be merged? Would like to be able to format currency for Ireland and in its current state I am not able using this library. This is fairly small update that should resolve the issue. Thanks.

**Comment by BenjaminVanRyseghem** [comment]
Sorry, I am **very** busy right now, I actually have an epic deadline next thursday...

I'll have a look as soon as I have 5' :smile:

Sorry for the inconvenience

**Comment by Blackbaud-WesGabbard** [comment]
@BenjaminVanRyseghem, can this PR be merged? Thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### 2.3.1
 
+- Fix issue with en-IE currency by adding currency position of `prefix`
```
Comment (BenjaminVanRyseghem) [inline CHANGELOG.md:7]: wrong format

File: AUTHORS [hunk 2]
```diff
@@ -62,3 +62,4 @@ Thomas Obermüller (https://github.com/thomas88)
 Tim Fish (http://github.com/timfish)
 Tim McIntosh
 Tim Schauder (https://github.com/cornflakeboy)
+Wes Gabbard (https://github.com/Blackbaud-WesGabbard)
```
Comment (BenjaminVanRyseghem) [inline AUTHORS:65]: Move it to another commit please, so that possibly reverting your code won't remove you from the authors list :smile:

**Review by BenjaminVanRyseghem — CHANGES_REQUESTED** [review]

**Comment by BenjaminVanRyseghem** [comment]
do you have any documentation showing that the currency is prefix? (to avoid another PR tomorrow to change it as it happened already :smile:) 

Sorry for the delay, and thanks for the PR

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-WesGabbard) [reply CHANGELOG.md]: Sorry about that, pushed change to add PR #

File: AUTHORS (hunk 2 above)
Comment (Blackbaud-WesGabbard) [reply AUTHORS]: Done

**Comment by Blackbaud-WesGabbard** [comment]
Hey @BenjaminVanRyseghem,
Understand completely. From my understanding the Euro sign placement depends on the country/language. English language countries put the sign before. 

Few references:
https://www.reddit.com/r/AskEurope/comments/23pu36/does_the_euro_sign_go_before_or_after_the_number/
https://www.quora.com/Does-the-euro-symbol-go-ahead-of-the-number-300-or-behind-it-300-I-always-write-it-the-second-way-but-Im-not-sure-Im-right
https://en.wikipedia.org/wiki/Language_and_the_euro#Summary

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 ### 2.3.1
 
+- Fix #569: Fix issue with en-IE currency
```
Comment (BenjaminVanRyseghem) [inline CHANGELOG.md:7]: Still missing the `Thanks to ...` at the end :smile:

**Review by BenjaminVanRyseghem — CHANGES_REQUESTED** [review]

**Comment by BenjaminVanRyseghem** [comment]
Thanks for the correction.

Could you please rebase your commits so that there are only 2:
- one with the code and the changelog
- one with the AUTHORS

thanks!

**Comment by BenjaminVanRyseghem** [comment]
One test failed:

```
 en-IE formats currency correctly

    Expected '-€1000.23' to be '€-1000.23', 'Should format currency correctly -1000.234 with $0.00'.
```

**Comment by Blackbaud-WesGabbard** [comment]
My bad, long day. I should have at least made sure the unit tests passed locally before pushing up.

**Comment by BenjaminVanRyseghem** [comment]
you removed `package-lock.json`. Could you put it back? :smile:

**Comment by stephenmcbride** [comment]
@Blackbaud-WesGabbard any chance you could re-visit this PR to make the changes requested? As an Irish dev I'm keen to see it merged 😄

**Comment by Blackbaud-WesGabbard** [comment]
@stephenmcbride @BenjaminVanRyseghem closing out and moving to https://github.com/BenjaminVanRyseghem/numbro/pull/597

**Pull request closed without merging**